packages/components/package.json

```diff
@@ -50,6 +50,7 @@
 		"@wordpress/rich-text": "file:../rich-text",
 		"@wordpress/warning": "file:../warning",
 		"classnames": "^2.2.5",
+		"construct-style-sheets-polyfill": "^2.4.16",
```

Comment:
If this polyfill is too much, we can as well avoid using `(new CSSStyleSheet()).replaceSync`. For example, we can just stamp styles in `<style>` tags and reference them here, or make some build flow to construct those our of `.(s)css` files.